handlers.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+var http = require("http");
+
+exports.callSusiApi = (slots, session, response) => {
+    let query = slots.query.value,
+        viewCount;
+
+    if(query === "help"){
+        http.get("http://api.susi.ai/susi/chat.json?q="+"Start+chatting", (response1) => {
+            var body = "";
+            response1.on("data", (chunk) => { body += chunk; });
+            response1.on("end", () => {
+                var data = JSON.parse(body);
+                viewCount = data.answers[0].actions[0].expression;
+                response.say(viewCount,false);
+            });
+        });
+    }
+    else if (query === "stop" || query === "cancel"){
+        viewCount = "Ok, Susi.AI chatbot exiting";
+        response.say(viewCount,true);
+    }
+    else{
+        var endpoint = "http://api.susi.ai/susi/chat.json?q="+query; // ENDPOINT GOES HERE
+
+        http.get(endpoint, (response1) => {
+            var body = "";
+            response1.on("data", (chunk) => { body += chunk; });
+            response1.on("end", function(){
+                var data = JSON.parse(body);
+                if(data.answers[0].actions[1]){
+                    if(data.answers[0].actions[1].type === "rss"){
```

Comment:
This can be done in one if else with && Codacy is showing that complexity is high here.

Reply:
@AliAyub007 I have tried it, the warning is still there. Thanks.